index.js

```diff
@@ -1,11 +1,29 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(number){
+  let result
+  if( number < 400 ) {
+    result = 'This one is on me!';
+  } else if (number > 400 && number < 2000 ) {
```

Comment:
You don't need to include the `> 400` bit since the first part of the conditional covers that. 
```suggestion
  } else if (number < 2000 ) {
```